### ISSUE TITLE
[Design] Logged-in homepage name tag section

### ISSUE THREAD

**Issue by mshea0606** [department-of-veterans-affairs/va.gov-team#13918]
## Background
See [Design plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/logged-in-homepage/2.0-redesign/product/LIH-outline-and-timeline.md)

We do not need to follow what is currently in mockups, but it's a place to start. Currently in mockups we are showing:

User Name
User Military Branch
Go to profile [Link]

Disability Rating
Go to disability [Link to claims]

Number of Claims in progress
Go to claims [Link to claims]

Appointment date/doctor (this one especially needs to be thought through)
Go to appointments [Link to VAOS]

<img width="643" alt="Screen Shot 2020-09-23 at 2 28 23 PM" src="https://user-images.githubusercontent.com/58752772/94065984-50c27900-fd98-11ea-91bb-ff6ce6626919.png">

## Tasks
- [x] Determine what information needs to be included in this overview
- [x] Visual design exploration for users with all information included
- [x] Visual design exploration for empty states
- [ ] Work with copy to approve messaging
- [x] High fidelity mockups
- [x] Review with Samara/Matt
- [ ] Appropriate stakeholder reviews and approvals have been completed

**Comment by Samara-Strauss** [comment]
Curious as to what this is. @tressaellen or @callen2563 — can you clarify?

**Comment by tressaellen** [comment]
@Samara-Strauss this was intended for an 'overview' section of the page, basically any personalized high level important info that a user might want to see.  This has translated into that nametag portion of the page, so we could probably change the title to something closer to that.

**Comment by Samara-Strauss** [comment]
Got it! Thanks for letting me know, @tressaellen.

**Comment by Samara-Strauss** [comment]
And yes, +1 to grooming this ticket to more clearly reflect the actual work planned.

**Comment by tressaellen** [comment]
@Samara-Strauss I've updated this ticket, let me know if you think it's missing anything.

**Comment by Samara-Strauss** [comment]
I changed the name of this ticket to "[Design] Logged-in homepage overview section" since this overview/new name tag area will show regardless of whether people have health care or not (though we obviously wouldn't show the disability/claims/appointment info if those boxes didn't apply to somebody).

Otherwise, the tasks list looks good to me!

**Comment by tressaellen** [comment]
Thank you! I was struggling with what to call it, that sounds good to me.

**Comment by Samara-Strauss** [comment]
Also, an idea that you can take or leave: for appointments, maybe that area shows something like "2 upcoming" instead of a date and doctor name. We could then pull out more detailed appointment info on the dashboard itself.

**Comment by tressaellen** [comment]
Ooh yes - I like that idea, I'll mock it up. It would make things more consistent with how the boxes are designed as well.

**Comment by tressaellen** [comment]
Here is a concept for users with all the information included:

![Screen Shot 2020-10-05 at 12.59.23 PM.png](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/6af8c370-8a05-440f-9b13-843009014343)

![Screen Shot 2020-10-05 at 4.42.46 PM.png](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/346ab68e-066f-43d1-b725-abf018fc8a6a)

Here is a concept for users with no information to show:

![Screen Shot 2020-10-05 at 12.59.30 PM.png](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/0244f711-5bf9-4474-922f-90beb86a6f3b)

![Screen Shot 2020-10-05 at 4.44.29 PM.png](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/90d0f002-6cd4-4b8b-b277-ab9002e57be3)

We're planning on re-evaluating once all the designs in the main content area are done, and decide if it's necessary to have all of that info in the header.

**Comment by Samara-Strauss** [comment]
This looks like a great start!

- I can't quite tell how wireframe-y this is vs. how done it is. What stage of fidelity am I looking at?
- Why aren't you showing the claims and appointments blocks on mobile? We don't want to lose that information.

**Comment by tressaellen** [comment]
@Samara-Strauss 

- This is high fidelity, I changed the background for that top bar from the dark grey because it clashes with the header when it's right up against it. And having buttons for each of those sections was really visually heavy, so we went with the links.
- I'm playing with a few options for that - with that much information in the header in mobile I'm nervous we'll run into the problem with it taking up a lot of screen real estate... we could do something like this, though I'm not sure if this would fly with accessibility.
<img width="349" alt="Screen Shot 2020-10-06 at 9 14 51 AM" src="https://user-images.githubusercontent.com/58752772/95221735-38912780-07a4-11eb-9e08-a197e9c59307.png">

**Comment by Samara-Strauss** [comment]
> This is high fidelity, I changed the background for that top bar from the dark grey because it clashes with the header when it's right up against it. And having buttons for each of those sections was really visually heavy, so we went with the links.

Ugh, COLOR! Why is it so hard?? I totally get that the dark gray clashes, but the light gray feels wireframe-y and undone to me. I realize some of that may be because the rest of the screen actually is still a wireframe, but are there other options/things we can do here? Do we need to see if someone like Ryan Thurlwell can help?

> I'm playing with a few options for that - with that much information in the header in mobile I'm nervous we'll run into the problem with it taking up a lot of screen real estate... we could do something like this, though I'm not sure if this would fly with accessibility.

I really like the treatment for the mobile view in the mockup with the disability rating/claim/appointment info stacked. I hear you on the screen real estate issue and the accessibility issue. Definitely check in with Jen re: accessibility. As for real estate, let's leave that up to user testing. I do hear what you're saying, but this is much more helpful info to users than seeing their military seal, which is my favorite design detail but I can totally understand why it felt like it was taking up too much room when we tested the profile header. However, if they think this is super important information, they'll think it's worth the real estate. You're smart to be thinking about it, and this is definitely something we should evaluate in testing.

**Comment by tressaellen** [comment]
> Ugh, COLOR! Why is it so hard?? I totally get that the dark gray clashes, but the light gray feels wireframe-y and undone to me. I realize some of that may be because the rest of the screen actually is still a wireframe, but are there other options/things we can do here? Do we need to see if someone like Ryan Thurlwell can help?

I think a big part of it is that the rest of the screen is undone... once we wrap up healthcare, I'll revisit this and see if we can make it look more polished with the rest of page. And I can also bring this up in the platform check-in on thursday.

**Comment by Samara-Strauss** [comment]
Sounds good to me. I think we may be able to resolve this with more eyes on the problem.

Keep up the great work!!

**Comment by Samara-Strauss** [comment]
Now that we are pivoting to a My VA redesign, I think we can go back to the dark gray background for the nametag area like we have in the profile. That's because My VA has a breadcrumb, so we'll have whitespace between the nav and header like we have right now. Let me know what you think about this!

<img width="1273" alt="Screen Shot 2020-10-08 at 10 59 18 AM" src="https://user-images.githubusercontent.com/34068740/95477067-f1e32f00-0955-11eb-9066-f6806469dda0.png">

**Comment by tressaellen** [comment]
You read my mind! We can still work on the treatment of the information displayed, but the dark bar works with the breadcrumbs. 

<img width="682" alt="Screen Shot 2020-10-09 at 12 26 22 PM" src="https://user-images.githubusercontent.com/58752772/95618658-eeb17700-0a19-11eb-8efb-7b8991741748.png">
<img width="224" alt="Screen Shot 2020-10-09 at 12 26 26 PM" src="https://user-images.githubusercontent.com/58752772/95618665-f07b3a80-0a19-11eb-8f04-d53dd2d1805a.png">

**Comment by Samara-Strauss** [comment]
It looks so good!

One thing — for the appointments section of the name tag, the data there should match the appointments block in the health care section. In this mockup, you are showing the facility name in the appointment section of the name tag, but in the appointments block in the health care section, you show doctor name and not facility. I would go with whatever you think users would expect to see most (we'll confirm assumptions in user testing), and be consistent in both places.

**Comment by tressaellen** [comment]
Yes, good point. This data is what shows in the mobile app - I was planning on asking them if they had tested that data yet. Regardless, we should do our own testing around it.

**Comment by Samara-Strauss** [comment]
> Regardless, we should do our own testing around it.

Agreed.

Let's be consistent at least in our own page, and then we can work with the mobile app team to determine whether facility or doctor (or both!) should be shown with appointment info.

**Comment by callen2563** [comment]
@Samara-Strauss @tressaellen Hey ladies, here are the latest mocks. I've added a table header on desktop and went though and cleaned up some areas and incorporated the latest feedback. Let me know if I missed any thing. Thanks!
https://cassandraallen738782.invisionapp.com/overview/HP-desktop-logged-in-ckf3qguc91olj019242g490ji/screens
https://cassandraallen738782.invisionapp.com/overview/HP-mobile-logged-in-ckeyhydog01kz01xd3tao5pcj/screens

**Comment by Samara-Strauss** [comment]
All of my feedback is in [this comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/13920#issuecomment-708414228) of the secure message ticket. Not going to repost here, so you can go there or [check the feedback doc](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/2.0-redesign/design-ia/samara-design-feedback.md#feedback-101420).

**Comment by callen2563** [comment]
@Samara-Strauss @rtwell @tressaellen Here is the updated invision link showing a mockup for the My VA tabbed version of overview. Let us know your thoughts. I'm not sure this is the right blue but can work on that in the mean time. Thanks!
https://cassandraallen738782.invisionapp.com/overview/HP-desktop-logged-in-ckf3qguc91olj019242g490ji/screens

**Comment by Samara-Strauss** [comment]
**Name tag**

- I really like the name tag in blue! Would love to also see an iteration without the tab and with the bread crumbs either in the name tag or below the name tag above the side navigation. We should have this in our back pocket depending on who makes final calls about updates to the navigation.

**Learn more about VA health benefits**

- I liked when this was on the same line as enrollment date. Can you move it back to that line but right align it to the opposite margin instead of having it left aligned under enrollment date? Let me know if this doesn't work for whatever reason. I realize it will all be stacked on mobile anyway, but that link can easily move responsively.

**Appointments**

- Can we rename this "Upcoming appointments"? Also, if we do that, we can remove the word "upcoming" in "disaplaying X upcoming appointments."
- Also, can we change the CTA to "View all appointments"? That will clue in the user that there is more than what is listed in the event that they have many appointments already scheduled.

**Prescriptions**

- Can we rename this section "Prescription refills"?
- "Active" is inside lingo on our end, so we should remove that word from "active prescriptions".

**Secure messages**

- Did y'all confirm we can actually detect which messages are new? Sorry if you already have, but I don't remember getting clarification on that. We should confirm (if you haven't already) before showing the word "new" there. Also, if you can detect and show what is "new", please use a lowercase "n" in the word.

**General**

- For the table headings, can you remove the colon: after each header label?

**Comment by tressaellen** [comment]
@Samara-Strauss, yep, Erik and Sandra confirmed that we can detect which messages are new. (I don't think we ever explicitly told you that)

**Comment by tressaellen** [comment]
Hi @Samara-Strauss @rtwell @callen2563 - here is the [updated invision link](https://cassandraallen738782.invisionapp.com/overview/HP-desktop-logged-in-ckf3qguc91olj019242g490ji/screens) for the latest round of designs. I incorporated Samara's feedback, updated the nametag area and added some info to the appointment cards.  For the appointment section of the nametag, I just showed the date and time and then relayed the rest of the info in the cards below, I'd love any thoughts on that. There's also three different options for displaying/not displaying breadcrumbs.

**Comment by rtwell** [comment]
question (and if this is too big of a question, we can table it for next week): I'm curious as to the reason(s) behind the persistent (I _think_ samara referred to it as the "profile area") across all 3 sections of My VA. I know that Claims, Disability, and Appointments are the top 3 items for Veterans, but when a user clicks on Health care, it seems like we could display specifically Health care content. I wonder how effective the Claims or Disability tiles are for a user at this point because they have elected to view their Health care page of My VA. I compare this to a banking app—when i have drilled down into my credit card, i do not see anything about my checking account or my savings or anything like that—it's a focused view on one subject. Have we considered a purely "dashboard" or "My VA home" page that houses just the important stuff (agnostic of benefit), and then each chunk of the dashboard links into the 3 big buckets?

Secondly, as we're developing the VA mobile app (which is _almost verbatim_ what we're doing with My VA) and updating VAOS, AND trying to keep my design system hat on, what if we try to develop a component-ized approach to some of these designs (or parts of them). For example, an appointment card/tile. Right now, the mobile app has a version, VAOS has a version, and My VA has a version, but they all share _exactly the same information_. VAOS will soon be updating it's card designs to reflect the app (closely, anyway), and I can totally see a case for My VA to follow suit. Thoughts? It will def take some wrangling but I suspect we could leverage what other teams have learned _and_ bring some visual consistency to the experience.

**Comment by tressaellen** [comment]
@rtwell - the original idea behind the "profile area/nametag" was to elevate the most important pieces of information based on the first round of user research... that's an interesting thought to have that change dynamically depending on what page is selected. I'm definitely down to explore that... we'd need to figure out what specific information to elevate for each section.  Also, we have dabbled with the idea in previous iterations of having a sort of 'dashboard' landing page, especially pre-side navigation when everything was going to be on one page.  I think it could potentially be worth looking into, though we'd need to explore how that would work for users who don't have any healthcare/benefits. Let's talk more about this on Monday.

I think streamlining/component-izing is a **great** idea, especially since they are all presenting the same information.

**Comment by Samara-Strauss** [comment]
> but when a user clicks on Health care, it seems like we could display specifically Health care content.

I think updating the name tag area for each section is an interesting idea that maybe we can talk through on Monday (or in another chat). I think we could user test what we have and get feedback and go from there, or maybe talk about a couple iterations of how a more dynamic treatment would work.

However, I do want to point out that veterans health care and disability info is very related to them — it's not a requirement all the time, but if you have a disability rating, you probably receive VA health care *at least* for that rating. Veteran's disability rating, as I understand from research, is one of their most important pieces of information at the VA. Yes, it's benefit related and certainly not agnostic, but it's something that can affect all sorts of benefits, not just your compensation payments. I could see maybe keeping that piece of information in the name tag even if there was some info we switched out (if we did that, changing info and static info would need to be treated different from a visual perspective so that changing information doesn't feel confusing).

> Have we considered a purely "dashboard" or "My VA home" page that houses just the important stuff (agnostic of benefit), and then each chunk of the dashboard links into the 3 big buckets?

There's not really important info that people would want to see up here, to the best of my knowledge, that's benefit agnostic. That's certainly something we can explore in research and iterate on over time, but I'm having trouble thinking of what information besides name/military branch we have access to and would put there.

> what if we try to develop a component-ized approach to some of these designs (or parts of them). For example, an appointment card/tile. 

Ditto to what Tressa said. I'm all for this.

**Comment by Samara-Strauss** [comment]
Here is my feedback on the latest iteration, [also captured in the feedback doc](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/2.0-redesign/design-ia/samara-design-feedback.md#feedback-101520).

**Name tag**

We already discussed most of this:

- Move link to profile back to the side nav.
- Continue to iterate on color treatment, since white text on this shade of blue does not pass accessibility standards.

**Health care overview**

- Not that I want us to waste space, but this feels a little too close to the name tag area. I think we need a little more white space between them.

**Lab and test results**

- Can you move this below Secure messages?
- This section is still not working at all. What else can we do here? Could we have a "You might also be interested in" section at the bottom of the page for links? Do we move this below secure messages and add some detail copy (heading > copy > CTA) to make this feel less weird? What else can you do with this link/CTA so it doesn't feel so random?

**General**

- Can we remove the divider lines between each section. I think we can just have the right amount of white space between sections to differentiate between them.

**Comment by rtwell** [comment]
**dumping this here as fodder for our discussion—please know I'm asking myself the same questions I'm asking you all:**

I suspect what’s happening here is more of UX problem than a visual design problem. There is tension between the profile portion of the page and the Healthcare portion of the page. Stepping back a moment: what job is the Health care page supposed to do? My guess is surfacing health care content relevant to them, and enabling them to take action (appointments, prescriptions, etc). 

Solving this problem from a mobile standpoint quickly brings this tension to light: as a user clicks “Health care” (or Disability or w/e) and, on page load, the user cannot see any of the Health care content (particularly anything actionable).

The dashboard only approach could solve this problem—My VA is an overview of a users VA stuff, and that is the initial view: here’s the really important stuff, at a glance. What’s a user to do with all of this important information? Since MVP will not have any notification-driven functionality, options for users is a bit limited: they have to parse the information in front of them and decide which benefit (I consider health care a benefit) they want to examine, maybe they want to check a prescription, we route them to health care, where they get an overview of health care, and can see prescriptions. 

Another reason I think a home/dashboard only page is effective: scalability. If we learn that Veterans want additional tiles for other benefits, we can continue to add without impacting the rest of the pages. 

I agree, disability and health care are related, just as claims are. That takes me back to my initial question: what is the job of this page? To help Veterans jump to different sections of My VA? Or focus Veterans on a particular task or topic? This is the tension I suspect is happening: both are fighting for the top slot on the page. 

Maybe an option to explore is quite similar to the mobile app approach. Perhaps My VA home is the big 3 tiles at the top, plus an actionable link to each section of My VA. I thought I would share the progress of the mobile app team for the [November launch](https://adhoc.invisionapp.com/share/YUYRT9BAPSB#/screens/431636866) (I think this is the most recent, not 100%). The missing layer in the mobile app is the thumbnail/overview content (disability rating, claims in progress, etc), but functionally we can see how the home page approach gives users a place to recenter, and drill down as needed. Are there reasons My VA should be any different? I know it’s not an app, and it’s not truly the same thing, but structurally and functionally, **they are so similar**. 

Additionally, that team is currently discussing what to call the app (literally, the name of the app in the App Store). What would you think about calling it “My VA”? MIght we leverage and interconnect the work we’re all doing—same visuals, same data/content, same calls to action—a more consistent experience for Veterans across devices and platforms!

**Comment by Samara-Strauss** [comment]
@rtwell if I'm picking up what you're putting down, I think this is a really good idea. Effectively, the extra info we have in the name tag area becomes an "Overview" section of the dashboard. I think this could be really awesome.

One thing — I do want to clarify that the goal of the name tag was not to be a "profile" on the dashboard. The goal was twofold:

- The original goal of the name tag is to send the message that this is "my" section, that this section is fundamentally different from the rest of the site. It communicates an extra level of personalization. While the name tag does also exist in the profile, it is not *the* profile.
- We added the "overview" info in an effort to have a quick, high level glance at your overall status. However, I hear you loud and clear that it's feeling busy/in competition with the content below it. I'm starting to feel the same way.

>I agree, disability and health care are related, just as claims are. That takes me back to my initial question: what is the job of this page? To help Veterans jump to different sections of My VA? Or focus Veterans on a particular task or topic? This is the tension I suspect is happening: both are fighting for the top slot on the page.

Thank you for asking this to bring this to the top of our minds. The goals of this page is to quickly get people to personally relevant tasks and information. That's it. That's the goal. While the homepage can quickly get people to tasks, it's not personalized, links may be irrelevant depending on the veteran, and it doesn't show them their personal information.

I think we have the right information to complete this goal. **However**, to your point, are we doing this effectively? Is our execution the right approach? I agree there is room for improvement.

> Additionally, that team is currently discussing what to call the app (literally, the name of the app in the App Store). What would you think about calling it “My VA”? MIght we leverage and interconnect the work we’re all doing—same visuals, same data/content, same calls to action—a more consistent experience for Veterans across devices and platforms!

We might be changing this to "Your VA" per content guidelines, but we have not formally had that conversation yet. I think "My VA" or "Your VA" works for both the dashboard and the app. The final name will just depend on what the content folks say.

**Comment by Samara-Strauss** [comment]
Excuse the poor quality of these sketches. @rtwell is this effectively what you're suggesting?

![My VA Overview_has data](https://user-images.githubusercontent.com/34068740/96271869-68f27600-0f9b-11eb-9408-6d243809581e.jpg)
![My VA Overview_no data](https://user-images.githubusercontent.com/34068740/96271871-698b0c80-0f9b-11eb-93e4-2cb82de79505.jpg)

**Comment by rtwell** [comment]
@Samara-Strauss yes! something like this!

**Comment by Samara-Strauss** [comment]
@rtwell I agree 100% I'm jealous I didn't think of this :) This feels like the absolute right direction to go in. I'm going to create a ticket to explore this next sprint.

**Comment by rtwell** [comment]
fresh eyes ❤️

**Comment by Samara-Strauss** [comment]
Added ticket for this work in #14952.

@tressaellen @callen2563 I think we can close this ticket out after y'all do the following:

- Remove the additional data from the name tag
- Keep the iterations on how we might handle the nav (tab) and bread crumbs.

Feel free to post screenshots here and then we can CLOSE!!

**Comment by sshein** [comment]
I actually think after chatting with me at one point, @callen2563 designed a version where "overview" was a part of the left nav, we just used the fancier left nav idea.  Maybe she can pull in some ideas from that version.

**Comment by tressaellen** [comment]
Ah!! :heart_eyes: I love this idea!! Then we're not running into the problem of trying to _cram_ everything into the nametag area... I got excited about it, so I mocked up something rough (I'll also drop it in #14952 ).
![My VA Overview](https://user-images.githubusercontent.com/58752772/96284542-2d9b7b80-0f8a-11eb-9186-2ee3aefc22b4.jpg)

Also, here's the screenshots for the healthcare section updates, and I'll go ahead and close this ticket! I added an 'additional information' section and included the lab and test results link there, we can discuss what other content might go there.
![Healthcare VA breadcrumb 3](https://user-images.githubusercontent.com/58752772/96284641-5754a280-0f8a-11eb-847b-d4c5de3ae6f8.jpg)

![Healthcare VA breadcrumb 1](https://user-images.githubusercontent.com/58752772/96284634-558adf00-0f8a-11eb-8a1e-7f1f2610da9a.jpg)
![Healthcare VA breadcrumb 2](https://user-images.githubusercontent.com/58752772/96284636-56bc0c00-0f8a-11eb-9731-1f7756bb83f8.jpg)
